packages/Select/src/Select.styles.js

```diff
@@ -1,19 +1,63 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
+import CaretDownIcon from "@paprika/icon/lib/CaretDown";
+import * as types from "./types";
 
-const svgString = color =>
-  `<svg color='${color}' width='18' height='32' xmlns='http://www.w3.org/2000/svg'><path fill='currentColor' d='M18.286 12.571q0 0.464-0.339 0.804l-8 8q-0.339 0.339-0.804 0.339t-0.804-0.339l-8-8q-0.339-0.339-0.339-0.804t0.339-0.804 0.804-0.339h16q0.464 0 0.804 0.339t0.339 0.804z'></path></svg>`;
+const sizeStyles = {
+  [types.SMALL]: css`
+    ${stylers.fontSize(-2)}
+    height: ${stylers.spacer(3)};
+  `,
+  [types.MEDIUM]: css`
+    ${stylers.fontSize(-1)}
+    height: ${stylers.spacer(4)};
+  `,
+  [types.LARGE]: css`
+    ${stylers.fontSize()}
+    height: ${stylers.spacer(5)};
+  `,
+};
+
+const placeholderStyles = ({ isDisabled }) => css`
+  ${stylers.placeholder}
+
+  ${!isDisabled &&
+    css`
+      &[disabled] {
+        color: inherit;
+      }
+    `}
+
+  option {
+    color: ${tokens.color.black};
+    font-style: normal;
 
-const selectArrow = (color = tokens.color.black) => `
-  background-image: url("data:image/svg+xml;base64,${btoa(svgString(color))}");
+    &:first-child {
+      color: inherit;
+      font-style: inherit;
+    }
+  }
 `;
 
-export const Select = styled.div`
+export const CaretIcon = styled(CaretDownIcon)(
+  ({ isDisabled }) => css`
+    color: ${isDisabled ? tokens.color.blackLighten60 : tokens.textColor.icon};
+    font-size: 19px;
```

Comment:
Ideally this would be 20px, but then the caret is actually bigger than in the mockups.  Probably the actual icon needs to be updated so the caret is smaller relative to its canvas size.

Reply:
This is why I updated the `CaretUp` and `CaretDown` icons 😏